.github/workflows/deploy-dev.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy (Dev)
+
+on:
+  push:
+    branches: dev
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: composer install
+    - name: Run tests
+      run: vendor/bin/phpunit
+
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.PUSH_TOKEN }}
+
+    - name: Deploy to production
+      uses: appleboy/ssh-action@master
+      with:
+        username: laravel
+        host: 161.35.32.131
```

Comment:
Could this be put in an env variable

Reply:
The hostname? Don't think it's necessary as the server IP is publicly visible anyway. Okay it's hidden by the domain but it's not exactly difficult to expose the actual server IP.

Reply:
More for if it needs to be changed for any reason